index.js

```diff
@@ -628,7 +623,7 @@ RedisClient.prototype.return_reply = function (reply) {
         command_obj = this.command_queue.shift();
     }
 
-    queue_len = this.command_queue.getLength();
+    queue_len = this.command_queue.length;
```

Comment:
I prefere writing `.length` as most queues use that and it is already implemented to support it.